script.js

```diff
@@ -224,3 +224,23 @@ form.addEventListener('submit', (e) => {
     formErrorMsg.innerHTML = 'success';
   }
 });
+form.addEventListener('change', () => {
```

Comment:
- [ ] As requested by the previous reviewer, When the user changes the content of any input field in the form the data object is updated and it is saved to the local storage. 🤔  Actually, I have to click on the `Get in touch` button for this to happen otherwise I get the previously saved input content on the page load. Kindly make sure When the user loads the page, the input fields are saved in the local storage and pre-filled with the data.